frontend/packages/data-portal/app/components/Breadcrumbs.tsx

```diff
@@ -77,32 +79,46 @@ export function Breadcrumbs({
   )
 
   return (
-    <div className="flex flex-row gap-sds-s text-sds-body-s leading-sds-body-s text-sds-gray-black items-center whitespace-nowrap content-start">
-      <Breadcrumb
-        text={t(variant === 'deposition' ? 'allDepositions' : 'allDatasets')}
-        link={browseAllLink}
-        className="shrink-0"
-      />
-      {chevronIcon}
-      {variant === 'deposition' ? (
-        <Breadcrumb text={t('deposition')} />
-      ) : (
+    <div className="flex flex-col flex-auto gap-1">
+      {previousDepositionId != null && variant !== 'deposition' && (
+        <Link
+          className="uppercase font-semibold text-sds-caps-xxxs leading-sds-caps-xxxs text-sds-primary-400"
+          to={`/depositions/${previousDepositionId}`}
```

Comment:
could you add filter carryback behaviour to this (or lmk if you plan on doing it in another PR)?